addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.js

```diff
@@ -11,9 +11,13 @@ class ChatterTopbar extends Component {
      */
     constructor(...args) {
         super(...args);
+        this.state = useState({
+            composerIsLog: null,
+        });
```

Comment:
not a good idea to sync 2 states. Prefer a single source of truth, so keep state on parent and pass it as props to topbar